boards/sodaq-sara-sff/Makefile.dep

```diff
@@ -0,0 +1,3 @@
+USEMODULE += boards_common_sodaq
```

Comment:
This line should be removed.

Reply:
Yeah, I forgot that one. It should be OK now.